prism/src/test/java/org/apache/falcon/jdbc/MonitoringJdbcStateStoreTest.java

```diff
@@ -85,66 +86,72 @@ public void init() {
 
     @Test
     public void testInsertRetrieveAndUpdate() throws Exception {
```

Comment:
Update the test to use some processes too, to ensure your changes are well tested.